gitignore patch

Copyright (C) 2014 david_yang <david_yang@accton.com>

SPDX-License-Identifier:     GPL-2.0

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..f7775cf
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,7 @@
+*.o
+*.patch
+sys_eeprom/onie-syseeprom
+tools/i2cdetect
+tools/i2cdump
+tools/i2cget
+tools/i2cset
